docs/core/tools/dotnet-install-script.md

```diff
@@ -100,8 +100,8 @@ The install scripts do not update the registry on Windows. They just download th
 
   Specifies the source channel for the installation. The possible values are:
 
-  - `Current` - Most current release.
-  - `LTS` - Long-Term Support channel (most current supported release).
+  - `STS` - Standard Term Support SDK
```

Comment:
There should be no "SDK" after Standard Term Support

Reply:
I'll change it, but note that this was taken verbatim from the scripts themselves.